data/json/items/gun/nail.json

```diff
@@ -24,8 +24,7 @@
     "reload": 50,
     "valid_mod_locations": [ [ "grip mount", 1 ], [ "rail mount", 1 ], [ "sights mount", 1 ], [ "stock mount", 1 ], [ "underbarrel mount", 1 ] ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "nail": 20 } } ],
-    "melee_damage": { "bash": 6 },
-    "flags": [ "WONT_TRAIN_MARKSMANSHIP" ]
```

Comment:
this reverts a fix

Reply:
I see, was patched while i doing barrel lenght at night

Reply:
Removing a bash damage also was intended?

Reply:
melee damage is still there even in your PR, but shows as a changed line due to comma - the ancient json spec used doesn't allow trailing commas, so you'll see changes like that when you change last line of json object